src/main/java/es/deusto/spq/client/gui/views/auth/LoginView.java

```diff
@@ -27,8 +41,8 @@
     private JButton btnLogin;
     private JButton btnRegister;
     private final JPanel panel_3 = new JPanel();
-    private Logger log;
-    private HotelManagementController controller;
+    private final Logger log;
```

Comment:
Why the sudden use of finals?

Reply:
Yeah sorry about that it was because i had the eclipse configured to automaticly use final when posible. i used it for doing other project and i didnt notice to change that.